nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/main/java/org/apache/nifi/cluster/coordination/http/endpoints/RuntimeManifestEndpointMerger.java

```diff
@@ -48,7 +49,7 @@ public NodeResponse merge(final URI uri, final String method, final Set<NodeResp
 
         final RuntimeManifestEntity responseEntity = clientResponse.getClientResponse().readEntity(RuntimeManifestEntity.class);
         final RuntimeManifest responseManifest = responseEntity.getRuntimeManifest();
-        final Set<Bundle> responseBundles = responseManifest.getBundles() == null ? Collections.emptySet() : new LinkedHashSet<>(responseManifest.getBundles());
+        final Set<Bundle> responseBundles = responseManifest.getBundles() == null ? new HashSet<>() : new LinkedHashSet<>(responseManifest.getBundles());
```

Comment:
It seems like this should be `new LinkedHashSet` instead of `new HashSet` to return the same type of Set implementation in all circumstances. I will defer @markap14 either way.
```suggestion
        final Set<Bundle> responseBundles = responseManifest.getBundles() == null ? new LinkedHashSet<>() : new LinkedHashSet<>(responseManifest.getBundles());
```

Reply:
I think either is fine. I have no problem using LinkedHashSet to keep the implementation the same.